internal/engine/k8srollout/podmonitor_test.go

```diff
@@ -18,6 +19,10 @@ import (
 	"github.com/windmilleng/tilt/pkg/model"
 )
 
+// NOTE(han): set at runtime with:
+// go test -ldflags="-X 'github.com/windmilleng/tilt/internal/engine/k8srollout.PodmonitorWriteGoldenMaster=1'" ./internal/engine/k8srollout
+var PodmonitorWriteGoldenMaster = "0"
```

Comment:
I really like this Golden Master testing method that @jazzdan implemented a while back, because it makes testing log output SO much less painful. Originally it was just **pipeline_state_test.go**, then I added it to **logstore_test.go** and now here. So the assertSnapshot function is duplicated in three places. Should / can this be more DRY?